Quaternion/Quaternion.py

```diff
@@ -691,6 +691,18 @@ def dq(self, q2=None, **kwargs):
             q2 = Quat(q2, **kwargs)
         return self.inv() * q2
 
+    def __setstate__(self, state):
+        # this method is called by the pickle module when unpickling an instance of this class.
+        # It receives the dictionary of the unpickled state and must update the __dict__ of this
+        # instance. This is the place where we "upgrade" pickled instances of older versions of
```

Comment:
Should say `versions before 3.5.0` not `older`.

Reply:
is "earlier versions than 3.5.0" ok?

Reply:
actually, the first comment, before the if statement, was meant as a general comment. It would not change with newer versions. I will add the specific version inside the if statement.